bot/exts/recruitment/talentpool/_review.py

```diff
@@ -44,37 +46,93 @@
+
+        Returns a boolean representing whether a new vote was sent or not.
+        """
+        if not await self.is_ready_for_review():
+            return False
+
+        user = await self.get_user_for_review()
+        if not user:
+            return False
+
+        await self.post_review(user, True)
+        return True
+
+    async def is_ready_for_review(self) -> bool:
+        """
+        Returns a boolean representing whether a new vote should be triggered.
+
+        The criteria for this are:
+         - The current number of reviews is lower than `MAX_ONGOING_REVIEWS`.
+         - The most recent review was sent less than `MIN_REVIEW_INTERVAL` ago.
+        """
+        voting_channel = self.bot.get_channel(Channels.nomination_voting)
+
+        review_count = 0
+        is_first_message = True
+        async for msg in voting_channel.history():
+            # Try and filter out any non-review messages. We also only want to count
+            # one message from reviews split over multiple messages. We use fixed text
+            # from the start as any later text could be split over messages.
+            if not msg.author.bot or "for Helper!" not in msg.content:
+                continue
+
+            if is_first_message:
+                if msg.created_at > datetime.now(timezone.utc) - MIN_REVIEW_INTERVAL:
+                    log.debug("Most recent review was less than %s ago, cancelling check", MIN_REVIEW_INTERVAL)
+                    return False
+
+                is_first_message = False
 
-    async def reschedule_reviews(self) -> None:
-        """Reschedule all active nominations to be reviewed at the appropriate time."""
-        log.trace("Rescheduling reviews")
-        await self.bot.wait_until_guild_available()
+            review_count += 1
 
+            if review_count >= MAX_ONGOING_REVIEWS:
+                log.debug("There are already at least %s ongoing reviews, cancelling check.", MAX_ONGOING_REVIEWS)
+                return False
+
+        return True
+
+    async def get_user_for_review(self) -> Optional[int]:
+        """
+        Returns the user ID of the next user to review, or None if there are no users ready.
+
+        Users will only be selected for review if:
+         - They have not already been reviewed.
+         - They have been nominated for longer than `MIN_NOMINATION_TIME`.
+
+        The priority of the review is determined by how many nominations the user has
+        (more nominations = higher priority).
+        For users with equal priority the oldest nomination will be reviewed first.
+        """
+        possible = []
         for user_id, user_data in self._pool.cache.items():
-            if not user_data["reviewed"]:
-                self.schedule_review(user_id)
+            if (
+                not user_data["reviewed"]
+                and isoparse(user_data["inserted_at"]) < datetime.now(timezone.utc) - MIN_NOMINATION_TIME
```

Comment:
It's not a mistake or anything but I think `now - inserted_at > min_time` is easier to read. Ignore if you want.